src/actions/MastodonModelAction.C

```diff
@@ -47,7 +47,8 @@ validParams<MastodonModelAction>()
   params.addParam<bool>("dynamic_analysis", true, "false, if static analysis is to be performed.");
   params.addParam<Real>("beta", 0.25, "beta parameter for Newmark time integration.");
   params.addParam<Real>("gamma", 0.5, "gamma parameter for Newmark time integration.");
-  params.addParam<unsigned int>("dim", 3, "Dimension of the mesh.");
+  MooseEnum dim("1 2 3", "3");
```

Comment:
If you do: "1=1 2=2 3=3" then you do not need to to _dim + 1.

Reply:
The left side is the enum string the right side is the enum integer value.

Reply:
Oh, I was wondering how to do that. Good to know!